music/demos/visualizer.html

```diff
@@ -50,21 +51,148 @@ <h1>Visualizers</h1>
       <input id="tempoInput" value="100" type="range" min="20" max="240" value="120" step="1">
       <span id="tempoValue"></span>
     </span>
-  </section>
 
-  <h2>mm.PianoRollSVGVisualizer</h2>
-  <section>
+    <h3>mm.PianoRollSVGVisualizer</h3>
+    <div class="visualizer-container" id="container1">
+      <svg id="piano-roll-svg-viewer"></svg>
+    </div>
+
+    <h3>mm.StaffSVGVisualizer</h3>
     <div class="visualizer-container" id="container2">
-      <svg id="svg"></svg>
+      <div id="staff-svg-viewer"></div>
+    </div>
+
+    <h3>mm.PianoRollCanvasVisualizer</h3>
+    <div class="visualizer-container" id="container3">
+      <canvas id="piano-roll-canvas-viewer"></canvas>
     </div>
   </section>
 
-  <h2>mm.PianoRollCanvasVisualizer</h2>
   <section>
-    <div class="visualizer-container" id="container">
-      <canvas id="canvas"></canvas>
+    <h2>mm.StaffSVGVisualizer advanced usage</h2>
+    <p>
+      mm.StaffSVGVisualizer extends the basic configuration capabilities to add:
+      <ul>
+        <li>
+          Displaying the score in a <b>compact visualization</b> way, just like paper printed 
+          scores, by setting config.pixelsPerTimeStep to zero. Otherwise <b>proportional 
+          visualization</b> mode will be used like in mm.PianoRollSVGVisualizer, where notes 
+          are separated according with their duration from pixel 0 at leftmost side. In order
+          to get staff context information, proportional visualization makes music signatures 
+          blink sometimes to show hidden musical symbols. You will notice bar lines are pretty
+          close to first bar note, because there is where bar begins. You can enjoy padding bars 
+          in compact visualization mode.
+        </li>
+        <li>
+          <b>Automatic Clef detection</b> will be done according to the average note, being it 
+          F clef if it is under middle C and G clef in any other case. 
+        </li>
+        <li>
+          <b>Default Key declaration</b> to adapt the score to the right accidentals in case 
+          key is not stated for time 0 on the NoteSequence (otherwise, NoteSequence will 
+          overwrite it).
+        </li>
+        <li>
+          <b>Instrument Filtering</b> to draw a subset of the NoteSequence tracks in one single 
+          score. No filter will draw all tracks altogether in one score.
+        </li>
+        <li>
+          Follow scoreplaying with two added auto-scroll modes, besides default pagination  
+          (<b>ScrollType.PAGE</b> or void), like centering active note on the middle of the 
+          score in a note by note basis (<b>ScrollType.NOTE</b>) or centering it in bar sized 
+          packed chunks (<b>ScrollType.BAR</b>) every time activeNote is the first one of a
+          bar.
+        </li>
+        <li>
+          Multiple <b>key signature</b> and <b>time signature</b> are supported (but notice 
```

Comment:
I would add this as a comment and not include it in the demo, then. Most of the visualizations Magenta users do are of MIDI files, and if this is not automatic then it's not really useful to call it out as a demo. You can call it out as a feature in the class docs if you want, but I think it's very confusing to have it here.